website/addons/evernote/static/widget-cfg.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
```

Comment:
...and likely replaced with an `evernoteFangornConfig.js` file that does a few overrides to make this behave as read-only.